Driver/Driver/Custom/CustomXibView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CustomXibView.swift
+//  YonetimSistemi
+//
+//  Created by Fatih Doğan on 14.02.2023.
+//
+
+import UIKit
+
+class CustomXibView: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    private func commonInit (){
+        let view = loadViewFromNib()
+        view.frame = self.bounds
+        view.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```

Comment:
please use programmatic layout with anchors instead of using autoresizing way. Some of cases that I faced before, autoresizing has a side effect.